### PR TITLE
make equipment handlers use new i2c driver

### PR DESCRIPTION
**Pull request by Taylorr82**

### CODE REVIEW

File: equipment_handler/source/ina226.c [hunk 1]
```diff
@@ -49,40 +48,7 @@ int INA226_RegisterSet(i2cBASE_t *i2c,
...
-      //}
-//  }
-//  else
-//  {
-      //Busy is busy return a error
-//      error = -1;
-//  }
-  //return((int)I2C_getStatus(i2c));
-  return i2cIsBusBusy(i2cREG2);
+  return i2c_Send(i2c, addr, 2, &data);
```
Comment (jlazzz) [inline equipment_handler/source/ina226.c:51]: I assume this is defined in i2c_io.h? I looked and wasn't able to find that file, so can't really finish the review unless I see that code. Side note: i2c_Send and i2c_Receive sound very similar to i2cSend and i2cReceive and might be easily confused. Might suggest making the name a bit more verbose. These new functions use interrupts, right?

Comment (arrooney) [reply equipment_handler/source/ina226.c:51]: Check out https://github.com/AlbertaSat/ex2_obc_software/pull/50 (ideally this code would be linked as a static library, but alas).

These functions should replace the existing ones, and I'd prefer not making the names more verbose (the high level function name shouldn't/needn't reveal implementation details). If there are still i2c send/receive functions we should probably cut them out.

Comment (Taylorr82) [reply equipment_handler/source/ina226.c:51]: I agree the names can be confusing. What would you suggest for a better name?

Comment (jlazzz) [reply equipment_handler/source/ina226.c:51]: Perhaps something like i2c_Send_msg and i2c_Receive_msg or i2c_msg_Send and i2c_msg_Receive?

Thanks for pointing me to that other PR. Implementation looks good

File: equipment_handler/source/ina226.c [hunk 2]
```diff
@@ -109,58 +75,15 @@ int INA226_RegisterGet(i2cBASE_t *i2c,
                          INA226_Register_TypeDef reg,
                          uint16_t *val)
 {
+
   uint8_t data[2];
 
+  if (i2c_Send(i2c, addr, 1, &reg) == -1) {return -1;}
```
Comment (jlazzz) [inline equipment_handler/source/ina226.c:81]: Maybe I'm a noob, but tbh I really am not a fan of having this as a single line!

**Review by jlazzz — COMMENTED** [review]
See comments on individual lines.

File: equipment_handler/source/ina226.c (hunk 2 above)
Comment (Taylorr82) [reply equipment_handler/source/ina226.c:81]: I'm going to leave this for now and make a new pr with clang-format configured, after we settle on a configuration in https://github.com/AlbertaSat/ex2_obc_software/pull/49

**Pull request merged**